bundle.sh

```diff
@@ -50,9 +50,14 @@ mv gpodder.ui.tmp "$APP_PREFIX"/share/gpodder/ui/gtk/gpodder.ui
 # localization of Quit and other menu items controlled by gtk-mac-integration
 cp -R "$JHBUILD_PREFIX"/share/strings/*.lproj "$APP_PREFIX"
 
+# fixes this error at startup:
+# gPodder.app/Contents/Resources/lib/python2.7/site-packages/gpodder/gtkui/draw.py:300: GtkWarning: Cannot open pixbuf loader module file 'gPodder.app/Contents/Resources/etc/gtk-2.0/gdk-pixbuf.loaders': No such file or directory
+"$JHBUILD_PREFIX/bin/gdk-pixbuf-query-loaders" > "$APP_PREFIX"/etc/gtk-2.0/gdk-pixbuf.loaders
```

Comment:
This breaks gPodder.app on other computers because it lists absolute paths in `gdk-pixbuf.loaders`.
The correct file is generated by gtk-mac-bundler in `$APP_PREFIX/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache`,

The fix is then to reference this file instead of `$APP_PREFIX/etc/gtk-2.0/gdk-pixbuf.loaders`:
`launcher.sh:
export GDK_PIXBUF_MODULE_FILE="$bundle_lib/gdk-pixbuf-2.0/2.10.0/loaders.cache"`
I'll commit this after merging